### ISSUE TITLE
Maximum elongation, triangularity and wall gap

### ISSUE THREAD

**Issue by jonmaddock** [ukaea/PROCESS#325]
In GitLab by @mkovari on Sep 3, 2015, 10:45

Freidberg, Cerfon and Lee have written two papers on "Tokamak elongation - how much is too much?"  
(I: Theory:  http://arxiv.org/ftp/arxiv/papers/1508/1508.06658.pdf and  
II: Numerical results: http://arxiv.org/ftp/arxiv/papers/1508/1508.06664.pdf)

The second of these gives detailed results for the maximum possible elongation, for a range of values of aspect ratio, triangularity, wall gaps and a resistive wall parameter.  For example:  
![Capture](http://git.ccfe.ac.uk:8080/process/process/uploads/4e064a2888ecb28c57b3be52171a370e/Capture.PNG)

Perhaps we could fit some formulae to these curves.  

He has determined the optimum triangularity for maximum elongation, so we would have to do likewise.  We would also need to clarify what he is assuming about the continuity of the wall.  The variation in max elongation is not huge: for example, going from aspect ratio = 2 to 4, the elongation changes from about 2.35 to 2.0.  Of course the elongation increases substantially at very small aspect ratio, so we might consider using these results in the ST model.  The resistive wall parameter is given by the product of the growth rate of the vertical instability and the wall diffusion time.  It doesn't seem to have anything to do with the vertical control system.  Quoting Freidberg et al:
>A simple but reliable definition of the feedback parameter is based on the following
two observations. First, practical feedback is feasible when the growth rate γ of the
n = 0 vertical instability is small. Since the plasma is surrounded by a finite
conductivity wall the instability of interest is actually a resistive wall mode. The second
observation is that if the feedback coils are located outside the resistive wall, as they
usually are, then an effective feedback system must have a rapid field diffusion time τw .
This is important because once an unstable plasma motion is detected the feedback
response fields must quickly diffuse through the wall in order to reach the plasma. In
other words τw should be short.  

>In principle it is possible to trade off growth rate in favor of response time or vice versa.
However, the overall effectiveness of the feedback system is dependent upon the
combined smallness of γ and τw . The conclusion is that a simple parameter that takes
into account the feedback system is the product of these two quantities:  
γτw = feedback capability parameter

>Consequently, the engineering value of γτw is a
good measure of the feedback capability. It represents the maximum value of the
resistive wall growth rate that can be feedback stabilized.

In other words, the field diffusion time not only slows down the vertical instability of the plasma, but also slows down the response of the active control system.  As he doesn't actually calculate the growth rate, it's far from obvious how the combined parameter depends on wall conductivity.  On the other hand, it is clear from his results that a bigger wall gap permits less elongation.

Also, it seems to follow that the speed of the vertical control coils is not important, provided it is good enough to be limited by the diffusion through the wall.  But what about the size, location and power capability of the control coils?  Surely they will have some effect?

Not sure if it is feasible to incorporate this in the near future.
